autocompletefilter/filters.py

```diff
@@ -31,20 +32,26 @@ def get_admin_namespace():
         return request.resolver_match.namespace
 
     def get_url(self):
-        model = self.field.related_model
-        args = (
-            model._meta.app_label,
-            model._meta.model_name,
-        )
+        # Try to get URL for Django 3.2+
+        try:
+            return reverse("admin:autocomplete")
+        except NoReverseMatch:
+            pass
```

Comment:
It's better to check version explicitly. 
It's close to what's described here https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

Reply:
You're right. Already doing it like this in admin.py so this should be in the same style. Thanks!